services/slam/collector.go

```diff
@@ -0,0 +1,61 @@
+package slam
+
+import (
+	"context"
+
+	"go.opencensus.io/trace"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"go.viam.com/rdk/data"
+	"go.viam.com/rdk/spatialmath"
+)
+
+type method int64
+
+const (
+	getPosition      method = iota
+	getPointCloudMap method = iota
+)
+
+func (m method) String() string {
+	switch m {
+	case getPosition:
+		return "GetPosition"
+	case getPointCloudMap:
+		return "GetPointCloudMap"
+	}
+	return "Unknown"
+}
+
+// Position defines the struct returned by the capturer for GetPosition.
+type Position struct {
```

Comment:
Instead of using this, can you directly use the GetPosition proto response type as the return type? That's what we're going to move towards soon, so can have one less thing to migrate if we do it here

Reply:
Done!